src/garrysmod/gamemodes/zombiesurvival/gamemode/cl_targetid.lua

```diff
@@ -106,9 +106,14 @@ function GM:DrawSigilTargetHint(ent, fade)
 	util.ColorCopy(color_white, colTemp)
 
 	draw.SimpleTextBlur("Sigil", "ZSHUDFontSmaller", x, y, colTemp, TEXT_ALIGN_CENTER)
-	y = y + draw.GetFontHeight("ZSHUDFontSmaller") + 0
+	local fontHeight = draw.GetFontHeight("ZSHUDFontSmaller")
+	y = y + fontHeight + 0
 
 	draw.SimpleTextBlur("Press E to teleport", "ZSHUDFontTiny", x, y, colTemp, TEXT_ALIGN_CENTER)
+
+	if MySelf:IsValid() and MySelf:Team() == TEAM_HUMAN and self:PlayerCanPurchase(MySelf) then
+    	draw.SimpleTextBlur(translate.Get("press_f2_for_the_points_shop"):gsub("%!", ""), "ZSHUDFontTiny", x, y + fontHeight / 2, colTemp, TEXT_ALIGN_CENTER)
```

Comment:
Be mindful that you're performing pattern matching in a render hook. This could be expensive. Fixing it may be a premature optimization, but that's why I declared this once outside of scope in my example.

The downside though is if a user switches their language, we may not have the up-to-date translation. 

Also, you are calculating the y-value math inline with the function here, but you're doing it separately for the first draw call.

Reply:
Removed the pattern matching. Leaving the `!` prefix doesn't affect how this text is rendered.

Reply:
> Leaving the `!` prefix doesn't affect how this text is rendered

I disagree. I think it's a matter of consistency, where 1 message ends in an exclamation and the other doesn't. If you're leaning towards leaving in the inconsistency, I'm fine with that.